sfdoc/publish/tasks.py

```diff
@@ -103,9 +103,12 @@ def _find_duplicate_urls(url_map):
 
 
 def _scrub_and_analyze_html(
-    html_file, path, article_image_map, images, url_map, problems
+    bundle, html_file, path, article_image_map, images, url_map, problems
 ):
     html = HTML(html_file, path)
+    if html.docset_id and html.docset_id != bundle.docset_id:
+        problems.append(
+            f"HTML ProductMapUUID {html.docset_id} does not match bundle ID, {bundle.docset_id} in {html_file}")
```

Comment:
@prescod have we confirmed this properly works when the topic results in multiple HTML files?

Reply:
Limited review implies the plugin at least sets a different ditamap UUID on reused topics.

Reply:
Not specifically. Probably the easiest way to test is to deploy to staging and ask Carol to test that. She says it is easy to redo that test.